content/docs/introducing-jsx.md

```diff
@@ -1,22 +1,22 @@
 ---
 id: introducing-jsx
-title: Introducing JSX
+title: पेश है JSX
 permalink: docs/introducing-jsx.html
 prev: hello-world.html
 next: rendering-elements.html
 ---
 
-Consider this variable declaration:
+इस परिवर्तनशील घोषणा पर विचार करें::
 
 ```js
-const element = <h1>Hello, world!</h1>;
+const element = <h1>नमस्ते दुनिया!</h1>;
 ```
 
-This funny tag syntax is neither a string nor HTML.
+यह अजीब टैग सिंटैक्स न तो एक स्ट्रिंग है और न ही HTML है।
 
-It is called JSX, and it is a syntax extension to JavaScript. We recommend using it with React to describe what the UI should look like. JSX may remind you of a template language, but it comes with the full power of JavaScript.
+इसे JSX कहा जाता है, और यह जावास्क्रिप्ट के लिए एक सिंटैक्स एक्सटेंशन है। UI कैसा दिखना चाहिए, इसके लिए हम React के साथ इसका उपयोग करते हैं  JSX आपको एक टेम्पलेट भाषा की याद दिला सकता है, लेकिन यह जावास्क्रिप्ट की पूरी शक्ति के साथ आता है।
 
-JSX produces React "elements". We will explore rendering them to the DOM in the [next section](/docs/rendering-elements.html). Below, you can find the basics of JSX necessary to get you started.
+JSX React "एलिमेंट्स" का उत्पादन करता है। हम उन्हें [अगले भाग](/docs/rendering-elements.html) में DOM को प्रस्तुत करेंगे। नीचे, आप आरंभ करने के लिए JSX की मूल बातें पा सकते हैं।
```

Comment:
हम उन्हें [अगले भाग](/docs/rendering-elements.html) में DOM को प्रस्तुत करेंगे -> changes the meaning of the line

Reply:
How about this
`हम अगले भाग में उन्हें DOM में रेंडर करने के बारे में देखेंगे`

Reply:
cool